### ISSUE TITLE
pre-defined interface with fields: createdAt, updatedAt

### ISSUE THREAD

**Issue by playerx** [neo4j-graphql/neo4j-graphql#126]
It will be nice feature to have pre-defined interface, Node for example, which will have fields createdAt, updatedAt and createNode, updateNode mutations takes care to set its values automatically to nodes which will implement this interface. Thats something that I need to do manually for all objects

**Comment by jexp** [comment]
Yes good idea. but Node is already taken by GraphQL so it would probably be something like 
`Audited` 

Alternatively we could allow a user to specify their own interface and just annotate fields with an appropriate directive, which I would like more.
